mlrun/common/constants.py

```diff
@@ -38,6 +38,9 @@
 JOB_TYPE_RERUN_WORKFLOW_RUNNER = "rerun-workflow-runner"
 MLRUN_ACTIVE_PROJECT = "MLRUN_ACTIVE_PROJECT"
 
+MLRUN_AUTH_SECRET_PATH = "/var/mlrun-secrets/auth"
```

Comment:
```suggestion
MLRUN_JOB_AUTH_SECRET_PATH = "/var/mlrun-secrets/auth"
```


job/functions/everything that mlrun spins of curse but mainly for mlrun run pods *as opposed* to mlrun IDE or so.